### ISSUE TITLE
Keyboard shortcuts for layer management stop working after switching keyboard layout to RUS

### ISSUE THREAD

**Issue by kr1zo** [lbalazscs/Pixelitor#69]
**Pixelitor version:** 4.2.1
**Java version:** 1.8.0_221
**OS:** Windows 10 x64

**Severity:** S3 Major
**Priority:** P1 High
**Component:** FUNCTIONAL

**Summary:**
Keyboard shortcuts for layer management stop working after switching keyboard layout to RUS.

**Steps to reproduce:**
1. Run the Pixelitor.
2. Create a new image (Ctrl + N).
3. Create multiple layers on the image.
4. Switch to the RUS keyboard layout.
5. Press the keyboard shortcuts: Ctrl + ], Ctrl + [, Ctrl + Shift + ], Ctrl + Shift + [, Alt + ], Alt + [

**Expected result:**
Hotkeys should work on any keyboard layout.

**Link (screenshots/video):**
https://yadi.sk/i/KstzmM0DzWtoxg

**Additional Info:**
I know that today you fixed this bug for the keys Ctrl + Shift + S, Ctrl + C, Ctrl + I etc, but not for Ctrl + ], Ctrl + [, Ctrl + Shift + ], Ctrl + Shift + [, Alt + ], Alt + [

**Comment by lbalazscs** [comment]
Fixed in GitHub, it will be in the next released version. 

I could't get characters like [ and ] to work with any keyboard layout as menu shortcuts, so I replaced these shortcuts with new ones that don't use special characters (Ctrl+Alt+R, Ctrl+Alt+L, Ctrl+Alt+Shift+R, Ctrl+Alt+Shift+L, Ctrl+Shift+R, Ctrl+Shift+L). These menus are not so important anyway - in early versions of Pixelitor it was not possible to reorder layers with mouse drag-and-drop, so they were important then, but not anymore.
